build-support/docker/remote_execution/Dockerfile

```diff
@@ -1,22 +1,18 @@
 # Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
This is the same as the `tc_remote_execution_file`, simply renamed. GitHub is showing it as a diff instead.